pkgs/by-name/hy/hyfetch/package.nix

```
@@ -48,15 +34,29 @@ python3Packages.buildPythonApplication rec {
   postInstall = ''
     mv ./docs/neofetch.1 ./docs/neowofetch.1
     installManPage ./docs/hyfetch.1 ./docs/neowofetch.1
+
+    install -m 755 neofetch $out/bin/neowofetch
   '';
```

Sorry, I was looking more at the source and saw this:
```suggestion
    install -m 755 neofetch $out/bin/neowofetch
  ''
  ++ lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
    installShellCompletion --cmd hyfetch \
      --bash <($out/bin/hyfetch --bpaf-complete-style-bash) \
      --fish <($out/bin/hyfetch --bpaf-complete-style-fish) \
      --zsh <($out/bin/hyfetch --bpaf-complete-style-zsh)
  '';
```